core/src/main/java/org/jruby/ext/fiber/ThreadFiber.java

```diff
@@ -31,18 +31,21 @@
     public ThreadFiber(Ruby runtime, RubyClass klass) {
         super(runtime, klass);
     }
-    
+
     public static void initRootFiber(ThreadContext context) {
+        initRootFiber(context, context.getThread());
+    }
+
+    public static void initRootFiber(ThreadContext context, RubyThread currentThread) {
```

Comment:
Is this overload really necessary? The new signature is fine, but nobody except us should be calling initRootFiber. Just nuke the original.